docs/cpp-client-guide.md

```diff
@@ -40,6 +40,32 @@ $(call import-add-path, NAKAMA_CPP_SDK)
 $(call import-module, nakama-cpp-android)
 ```
 
+#### Initialize Nakama SDK
+
+For most NativeActivity projects, if you have an entry point like:
+
+```cpp
+void android_main(struct android_app* state) {
+```
+
+Add include:
+
+```cpp
+#include "nakama-cpp/platform/android/android.h"
+```
+
+Add the following code at the top of the `android_main` function:
+
+```cpp
+Nakama::init(state->activity->vm);
+```
+
+#### Shared library size
+
+Don't be afraid that nakama shared library are near 100 Mb in size. After building final apk it will be just few Mb.
```

Comment:
Replace this with:

!!! Note
     Be aware that the client shared library size is approximately 100MB. After final target device compilation, the size will be drastically reduced to only a few megabytes.